integration_tests/coverage_report/not-required-in-test-suite.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+throw new Error(`
+  this error should not be a problem because
+  this file is never required or executed
+`);
+
+// Flow annotations to make sure istanbul can instrument non ES6 source
+module.exports = function(  // eslint-disable-line no-unreachable
```

Comment:
oh you already did this. Sorry, was on an outdated diff. Nevertheless, please put the disable comment above line 16, not next to the code.